app/components/embedded_bindings/embed_text.tsx

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {PureComponent} from 'react';
+import {LayoutChangeEvent, ScrollView, StyleProp, TextStyle, View, ViewStyle} from 'react-native';
+
+import Markdown from 'app/components/markdown';
+import ShowMoreButton from 'app/components/show_more_button';
+import {Theme} from '@mm-redux/types/preferences';
+
+const SHOW_MORE_HEIGHT = 60;
+
+type Props = {
+    baseTextStyle: StyleProp<TextStyle>,
+    blockStyles?: StyleProp<ViewStyle>[],
+    deviceHeight: number,
+    onPermalinkPress?: () => void,
+    textStyles?: StyleProp<TextStyle>[],
+    theme?: Theme,
+    value?: string,
+}
+
+type State = {
+    collapsed: boolean;
+    isLongText: boolean;
+    maxHeight?: number;
+}
+
+export default class EmbedText extends PureComponent<Props, State> {
```

Comment:
could we reuse the `AttachmentText` class or do we foresee future changes that would impact/stop us from reuse?

Reply:
Same as with ActionButtonText.